### PR TITLE
fix: removing the old logos for OBF, OPF, OPFF

### PR DESCRIPTION
**Pull request by teolemon**
### What
- fix: removing the old logo for flavors

### Why
We now have a language independant logo for all flavors, and translators keep translating to logo that don't exist, provoking test failures

### Part of
- #7295

### CODE REVIEW

File: po/common/hi.po [hunk 1]
```diff
@@ -5600,7 +5600,7 @@ msgstr ""
 
 msgctxt "org_protect_data_note"
 msgid "Removing or changing the provided data will be possible only by experimented contributors on the web site."
-msgstr ""
+msgstr "Hi"
```
Comment (VaiTon) [inline po/common/hi.po:5603]: I don't think this is right

**Review by VaiTon — CHANGES_REQUESTED** [review]

File: po/openfoodfacts/af.po [hunk 2]
```diff
@@ -16,12 +16,12 @@ msgstr ""
 #. make sure the logo exists for your language, otherwise ask @teolemon how to create a translated logo
 msgctxt "logo"
 msgid "openfoodfacts-logo-en-178x150.png"
-msgstr ""
+msgstr "openfoodfacts-logo-af-178x150.png"
```
Comment (teolemon) [inline po/openfoodfacts/af.po:19]:
```suggestion
msgstr ""
```

File: po/openfoodfacts/af.po [hunk 3]
```diff
@@ -16,12 +16,12 @@ msgstr ""
...
 
 #. make sure the logo exists for your language, otherwise ask @teolemon how to create a translated logo
 msgctxt "logo2x"
 msgid "openfoodfacts-logo-en-356x300.png"
-msgstr ""
+msgstr "openfoodfacts-logo-af-356x300.png"
```
Comment (teolemon) [inline po/openfoodfacts/af.po:24]:
```suggestion
msgstr ""
```

**Comment by teolemon** [comment]
@VaiTon Can you validate ? Also, Please do not remerge l10n_main into this.

**Review by VaiTon — APPROVED** [review]

**Comment by alexgarel** [comment]
@teolemon why wasn't this branche squasehd and merged ?

**Pull request merged**